workitem/workitemtype_repository.go

```diff
@@ -75,6 +76,28 @@ func (r *GormWorkItemTypeRepository) Load(ctx context.Context, spaceID uuid.UUID
 	return &res, nil
 }
 
+// Exists returns true if a work item type exists with a given ID
+func (m *GormWorkItemTypeRepository) Exists(ctx context.Context, spaceID uuid.UUID, id uuid.UUID) (bool, error) {
```

Comment:
@kwk As long as we use it now, we should keep it. But if you know another reason to remove it...